holoviews/util/command.py

```diff
@@ -1,16 +1,28 @@
+#! /usr/bin/env python
```

Comment:
This is only for testing really. In setup.py this script is setup as an entry point which means it will just be available, regardless of this line.